src/lazy/encoder/binary/v1_1/flex_uint.rs

```diff
@@ -46,79 +45,49 @@ impl FlexUInt {
     ///          an appropriate error message if reading fails.
     #[inline(always)]
     pub fn read(input: &[u8], offset: usize) -> IonResult<FlexUInt> {
-        const COMMON_CASE_INPUT_BYTES_NEEDED: usize = 8;
-
+        const COMMON_CASE_INPUT_BYTES_NEEDED: usize = 4;
```

Comment:
🪧 The hot path below now uses per-size `const` method invocations. This was lowered from `8` to `4` to strike a balance between compiled method size (for inlining) and performance.